edk2toolext/invocables/edk2_platform_build.py

```diff
@@ -37,6 +37,10 @@ def GetName(self):
         ''' Get the name of the repo, platform, or product being build '''
         pass
 
+    def GetUefiBuilderClass(self):
+        ''' Returns the platform builder class that we want to use '''
```

Comment:
Class should be a string? Can you clarify the comment?

Reply:
This allows a setting file to return the class, not an object that we want to create to use as a uefi builder